### ISSUE TITLE
Sign in to audio.com: accessibility issues

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#10463]
Pressing `Tab` puts the Cancel button in focus, and from there, you cannot move focus to any other element of the dialog. Pressing `Tab` or `Shift+Tab` won't do anything.

**Comment by dozzzzer** [comment]
@embarc-gabriel on Windows, I see this runtime error:

<img width="1000" height="475" alt="Image" src="https://github.com/user-attachments/assets/47a7dcec-c83a-439f-acfe-75cffd77e722" />

**Comment by embarc-gabriel** [comment]
Closed by https://github.com/audacity/audacity/pull/10504

**Comment by embarc-gabriel** [comment]
@dozzzzer what triggered this error?

**Comment by dozzzzer** [comment]
@embarc-gabriel Launching the build from #10504 on Windows 10. That's all.
